lib/liquid/tags/ifchanged.rb

```diff
@@ -1,16 +1,16 @@
 module Liquid
   class Ifchanged < Block
-    def render(context)
+    def render_to_output_buffer(context, output)
       context.stack do
-        output = super
+        block_output = super(context, '')
```

Comment:
It isn't documented that `render_to_output_buffer` must return the buffer. It seems unnecessary to require that, unless we wanted a way to replace the buffer completely.